### ISSUE TITLE
Local folder for Specific data

### ISSUE THREAD

**Issue by Lithimlin** [ckulka/baikal-docker#39]
In my docker compose I'm trying to specify a local directory to mount into the container's `Specific` directory so I can backup it more easily. However, it does not work. Instead, docker creates a volume for me.

Here's my docker-compose:
```
  baikal:
    image: ckulka/baikal:nginx
    container_name: baikal
    restart: always
    volumes:
     - ./volumes/baikal/config:/var/www/baikal/config
     - ./volumes/baikal/data:/var/www/baikal/Specific
```
Mounting the `config` directory works just fine.
Has anyone encountered a similar issue?

**Comment by BobWs** [comment]
This works for me 
```
 - ./volumes/baikal/Specific:/var/www/baikal/Specific \
 - ./volumes/baikal/config:/var/www/baikal/config \
 ```

**Comment by Lithimlin** [comment]
With that I get the following error in the web interface:
```
The FOLDER containing the DB file is not writable, and it has to.
Please give write permissions on folder '/var/www/baikal/Specific/db'
```
Also can you explain why I have to name the directory I'm mounting `Specific` and can't just name it `data`?

**Comment by Lithimlin** [comment]
Strangely enough, when trying to revert my changes, the `data` directory was correctly mounted into the container. However, there's still the issue with the permissions.

**Comment by BobWs** [comment]
I guess the “name” shouldn’t matter. If you want to call it “data” I think that that would not be the problem.

The permissions for the folders `config` and `data/specific` Should be `33` for both `group and user` not `root or something else`

**Comment by Lithimlin** [comment]
The "name" shouldn't matter but when I tried naming it `data`earlier, it didn't work and instead docker created a new volume.

I assume the permissions should be applied recursively to sub-directories as well? Should it be `033` or `330`? The (working) `config` folder currently has `755`.
I have already adjusted the `group and user` by setting the one for the `data` directory to the same as the `config` one.
